caasp-grafana-image/root/usr/local/bin/entrypoint.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -e
+
```

Comment:
maybe here we can link to the [upstream code from grafana-docker](https://github.com/grafana/grafana-docker/blob/master/run.sh), just to have some history in case something breaks